WORKSPACE

```diff
@@ -12,13 +12,13 @@ load("@bazel_skylib//lib:versions.bzl", "versions")
 
 versions.check("3.4.1")
 
-load("@rules_ruby//ruby:deps.bzl", "rules_ruby_register_toolchains")
+load("@rules_ruby//ruby:deps.bzl", "register_ruby_toolchain")
 
-rules_ruby_register_toolchains([
-    "system",
-    "ruby-3.0",
-    "jruby-9.2",
-])
+register_ruby_toolchain("system_ruby")
```

Comment:
Could we make the repository rule separate and register the toolchains manually?

```
ruby_runtime(name = "ruby-3.0")

ruby_runtime(name = "jruby-9.2")

ruby_runtime(name = "system_ruby")

register_toolchains(
    "@system_ruby//:toolchain",
    "@ruby-3.0//:toolchain",
    "@jruby-9.2//:toolchain",
)
```

Adding `name=` to the repository rules makes it more obvious that this is registering a repository.

Making the two separate makes it clear where the repositories are actually being created. (If I read `register_ruby_toolchain()`, I assume it is analogous to `register_toolchains()`, which only registers a toolchain and does not create any repositories.

Reply:
hmm yea I agree that it's probably better to explicitly register the toolchains.  For the `name=` I'm not convinced.  If *both* `name` and `version` were specified it makes sense, but since those will typically be the same it just invites bugs due to mismatches.  But if we only pass 1 argument `name=` is kindof confusing, since it's *also* a version spec

Reply:
I split out the toolchain registration and clarified this behavior in some of the examples